stable_baselines/ppo1/pposgd_simple.py

```diff
@@ -297,8 +297,10 @@ def learn(self, total_timesteps, callback=None, seed=None, log_interval=100, tb_
                     lens, rews = map(flatten_lists, zip(*listoflrpairs))
                     lenbuffer.extend(lens)
                     rewbuffer.extend(rews)
-                    logger.record_tabular("EpLenMean", np.mean(lenbuffer))
-                    logger.record_tabular("EpRewMean", np.mean(rewbuffer))
+                    if len(lenbuffer) > 0:
```

Comment:
Minor: I was wondering, when `len(lenbuffer) > 0`, len(rewbuffer) must be > 0 no?
If so, we can simplify the check ;)

Reply:
True, it was more a matter of long term code stability. Will do the change.